g3doc/quick_reference.md

```diff
@@ -786,6 +786,11 @@ All other ops in this section are only available if `HWY_TARGET != HWY_SCALAR`:
     `sum[i] + a[4*i+3]*b[4*i+3] + a[4*i+2]*b[4*i+2] + a[4*i+1]*b[4*i+1] +
     a[4*i+0]*b[4*i+0]`
 
+*   `VW`: `{u,i}{16,32,64}`: `V`: `{u,i}{8,16,32}`, `D`, 
```

Comment:
Let's change this to just mention the constraints on V, because D and VW are derived from V
`V`: `{u,i}{8,16,32}`, `D = RepartitionToWide<DFromV<V>`, `VW = Vec<D>`.